features/TopBar/NavItems.tsx

```diff
@@ -89,14 +89,11 @@ export const NavItems: FC = () => {
           <NavItem active={router.pathname.startsWith("/info")}>info</NavItem>
         </Link>
       </Popover>
-      <a
-        href="https://feedback.pickle.finance/"
-        target="_blank"
-        rel="noopener noreferrer"
-        style={{ textDecoration: "none" }}
-      >
-        <NavItem>feedback</NavItem>
-      </a>
+      <Link href="https://feedback.pickle.finance/" passHref>
```

Comment:
This fixes the error in the console warning about an anchor tag nested within another anchor tag being invalid markup.